rportfolio/src/components/skillsList/SkillsList.jsx

```diff
@@ -0,0 +1,21 @@
+import "./skillsList.css";
+import  Skills  from "../skills/Skills"
+import { skills } from "../../data";
```

Comment:
Try and start using import alias so you can import your files like so `@/components/skills/Skills`